backend/src/RedisDriver.js

```diff
@@ -0,0 +1,29 @@
+const Redis = require( 'ioredis' );
+const redisAdapter = require( 'socket.io-redis' );
+const dotenv = require( 'dotenv' );
+dotenv.config( { path: '.env' } );
+
+class RedisDriver {
+	constructor( io ) {
+		this.io = io;
+		this._config = {
+			host: process.env.REDIS_HOST,
+			port: process.env.REDIS_PORT
+		};
+		this._client = this._connect( io );
+	}
+
+	get client() {
+		return this._client;
+	}
+
+	_connect( io ) {
+		const client = new Redis( this._config );
+		io.adapter( redisAdapter( this._config ) );
```

Comment:
I have a doubt about this. The driver looks like be a part of data access layer but `redisAdapter` is something like a message broker. So theoretically it should be two separate things.

In the other hand, I'm not sure if a separate class wouldn't be overcomplicated in this case (because AFAIR there was an idea to create also `InMemoryDriver`).

WDYT @czerwonkabartosz ?

Reply:
Ok, you have noticed it first :D

Reply:
Done, you can see it in `c8d99f6`.